### DISCUSSION TITLE
Welcome to RevBayes Discussions!

### DISCUSSION THREAD

**Discussion by bredelings** [revbayes/revbayes#601] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bredelings** [comment]
Hi, I'm Ben Redelings.  I work on Bayesian sequence alignment and phylogenetics.  My favorite bird in the peregrine falcon.

**Comment by thauffe** [comment]
Good idea, let's fill this board with life!

My name is Torsten Hauffe.
I am a RevBayes beginner interested in inferring diversification rates from the fossil record and the mode and pace of trait evolution for extant species.
